app/state/stateSvc.js

```diff
@@ -488,6 +490,21 @@ function stateSvc(
     return svc.config.storypins;
   };
 
+  svc.save_storyframes = storyframes => {
```

Comment:
This function name should use JS camelcase syntax.

Reply:
Converted to camelcase.